public/app/plugins/datasource/loki/components/LokiQueryField.tsx

```diff
@@ -76,7 +76,7 @@ export class LokiQueryField extends React.PureComponent<LokiQueryFieldProps, Lok
         >
           <div className="gf-form gf-form--grow flex-shrink-1 min-width-15">
             <MonacoQueryFieldWrapper
-              runQueryOnBlur={app !== CoreApp.Explore}
+              runQueryOnBlur={false}
```

Comment:
I think we can safely remove this prop and all related code, since it was only used for the on-blur run in dashboards.

Reply:
Done removed this prop for both the prometheus and loki variant